pkgs/applications/virtualization/quickgui/default.nix

```diff
@@ -1,67 +1,59 @@
-{ stdenvNoCC
+{ fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
 , lib
-, fetchurl
-, autoPatchelfHook
-, dpkg
-, wrapGAppsHook3
+, flutter
 , quickemu
 , zenity
 }:
-
-stdenvNoCC.mkDerivation rec {
+let
+  runtimeBinDependencies = [ quickemu zenity ];
+in
+flutter.buildFlutterApplication rec {
   pname = "quickgui";
-  version = "1.2.8";
-
-  src = fetchurl {
-    url = "https://github.com/quickemu-project/quickgui/releases/download/v${version}/quickgui_${version}-1_lunar1.0_amd64.deb";
-    sha256 = "sha256-crnV7OWH5UbkMM/TxTIOlXmvqBgjFmQG7RxameMOjH0=";
+  version = "1.2.10";
+  src = fetchFromGitHub {
+    owner = "quickemu-project";
+    repo = "quickgui";
+    rev = version;
+    hash = "sha256-M2Qy66RqsjXg7ZpHwaXCN8qXRIsisnIyaENx3KqmUfQ=";
   };
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
 
-  nativeBuildInputs = [
-    autoPatchelfHook
-    dpkg
-    wrapGAppsHook3
-  ];
-
-  buildInputs = [
-    quickemu
-    zenity
-  ];
-
-  strictDeps = true;
-
-  unpackCmd = "dpkg-deb -x $curSrc source";
-
-  installPhase = ''
-    runHook preInstall
-
-    mv usr $out
+  gitHashes = {
+    window_size = "sha256-XelNtp7tpZ91QCEcvewVphNUtgQX7xrp5QP0oFo6DgM=";
+  };
 
-    runHook postInstall
-  '';
+  # These things are added to LD_LIBRARY_PATH, but not PATH
```

Comment:
I don't think this comment is accurate here anymore but everything builds and works so the package is fine.

Reply:
Comment removed.